### DISCUSSION TITLE
Plans for Incorporating Vaccinations?

### DISCUSSION THREAD

**Discussion by cbird808** [neherlab/covid19_scenarios#849] [Ideas]
I think I posted about this once before, but the time has come that we need to be able to incorporate vaccinations into model projections.  Are there any plans for this?  The seroprevalence was a nice addition, but it's a 1-time setting and doesn't allow for future vaccinations.

I think the simplest implementation for C19Scenarios would be to mimic the mitigations input, with the ability to control percent vaccinated over use-defined date ranges.  The difference from mitigations is that it should move people from susceptible to recovered.

For now, I'm controlling the effect of vaccinations with mitigations, but it leaves too much of the population susceptible and can only work for so long.

The alternative at this juncture is to switch to a new modeling platform or hack c19s.

**Comment by andreasamsler** [comment]
Hi there, fyi: https://github.com/openZH/covid_19/issues/1550#issuecomment-772587753
